### PR TITLE
Feature/bl UI 4863 grade component

### PR DESCRIPTION
**Pull request by ArshdeepSingh3-eaton**
Fixes # .

#### Changes proposed in this Pull Request:

-Add [Grade Component](https://github.com/etn-ccis/blui-react-native-component-library/pull/429)
-
-

#### Screenshots / Screen Recording (if applicable)

-

#### To Test:

-

#### Any specific feedback you are looking for?

-

### CODE REVIEW

File: components/KitchenSink.tsx [hunk 1]
```diff
@@ -202,68 +203,102 @@ export const KitchenSink: React.FC = (): JSX.Element => {
...
+                        <Grades.d />
+                        <View style={{ padding: 5 }} />
+                        <Grades.dMinus />
+                        <View style={{ padding: 5 }} />
+                        <Grades.f />
+                        <View style={{ padding: 5 }} />
+                        <Grades.custom label="CS" size={40} color="#fff" backgroundColor="#000" />
+                        <View style={{ padding: 5 }} />
+                        <Grades.f label="E" />
+                    </Card.Content>
```
Comment (huayunh) [inline components/KitchenSink.tsx]: Can we use a list or some other flex views here?

**Review by huayunh — APPROVED** [review]
Still not a big fan of it, but this is just a showcase demo, and I think I was being too picky. I will give this the green light

**Comment by jvgreiner** [comment]
Not sure whats going on with the default App test here but getting this error. Maybe upgrade Jest to v29 ?

image

**Comment by joebochill** [comment]
> Not sure whats going on with the default App test here but getting this error. Maybe upgrade Jest to v29 ?

This looks like something else not related to the addition of the Grade...let's log a new issue to look into the tests.

**Pull request merged**